apps/calendar/test/marionette/week_view_test.js

```diff
@@ -313,12 +320,31 @@ marionette('week view', function() {
       assert.equal(editEvent.startTime, pad(time) + ':00:00', 'startTime');
       assert.equal(editEvent.endTime, pad(time + 1) + ':00:00', 'endTime');
     }
+  });
 
-    function isShowingAddEventLink() {
-      return week.element.scriptWith(function(el) {
-        return !!el.querySelector('.md__add-event');
-      });
-    }
+  suite('event retargeting tests', function() {
+    test('When tapping around an event we can create another event',
+    function() {
+      week.waitForHourScrollEnd();
+
+      week.waitForNoScroll();
+      week.scrollToHour(6);
+      week.tapDayHour({ day: 7, hour: 6 });
+
+      assert.ok(isShowingAddEventLink(), 'The `add-event` link is displayed');
+
+      // The "+" button should work with event retargeting.
+      week.tapDayHour({ day: 7, hour: 6, bottom: -1 });
```

Comment:
I'd extract that value in a variable. It makes easier to test the scroll function :smiley: